assets/components/forms/standardFields/input.scss

```diff
@@ -0,0 +1,10 @@
+@import './formFields';
+
+.component-input:not([type="radio"]) {
+  @include form-field;
+
+  height: 38px;
```

Comment:
does this have to be explicitly set? could it be a combo of `min-height`, `padding`, and `line-height`?